root/edit/components/ListHeader.js

```diff
@@ -174,15 +174,13 @@ const QuickLinksSection = ({
     );
   }
   return (
-    <td>
-      {quickLinks.reduce((accum: Array<React$Node>, link, index) => {
-        accum.push(link);
-        if (index < (quickLinks.length - 1)) {
-          accum.push(' | ');
-        }
-        return accum;
-      }, [])}
-    </td>
+    quickLinks.reduce((accum: Array<React$Node>, link, index) => {
```

Comment:
Not sure I understand why this works, since components that return arrays still need keys on the elements in the array (or at least they should). Try editing the `MyButton` example on https://react.dev/learn and change the `return (...)` to `return [...]`.

Reply:
I tried to debug this and I honestly can't tell if it's a React bug that the key warning isn't logged here -- it definitely logs it if you return an array elsewhere, so it seems like to intent is that these should have keys at least.